channel-manager-operations/availabilityBlock.md

```diff
@@ -126,14 +126,15 @@
 | `company` | [`Company`](../channel-manager-operations/reservations.md#company) object | optional | The company associated with the block. |
 | `notes` | `string` | optional | Notes for the block. |
 | `pickupType`| [`PickupType`](#pickupType) object | required | PickupType of the block. |
+| `releasedStrategy` | [`Release strategy`](../mews-operations/availabilityBlock.md#release-strategy) object | required | Release strategy of the availability block. |
```

Comment:
It's a little bit confusing to me because it says `releasedStrategy` (with a 'd'), is that because it has already been released, or is it a mistake? If it's already been released, perhaps the description should be something like "Release strategy used when the availability block was released".

Reply:
Ohh that's my mistake, `releaseStrategy` is the true one.
special thanks for the attention.